README.md

```diff
@@ -179,6 +179,46 @@ $response = $mj->post(Resources::$Parseroute, ['body' => $body]);
 
 ```
 
+
+## New !! Specificities of the version 3.1 !
+
+This version add a new way to construct the Client or you call. We add the possibility to add an array with for parameters on both Client creation or api call (please, note that each of these parameters are preset and are not mandatory in the creation or the call) :
```

Comment:
Some typos to fix:
- add -> adds
- api -> API (style, not typo)
- "you call" => your call
- ...